pkgs/by-name/co/conduktor-ctl/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "conduktor-ctl";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "conduktor";
+    repo = "ctl";
+    rev = "v${version}";
```

Comment:
`"refs/tags/v${version}"` is safer if there is ever a branch name matching a tag name

Reply:
Thank you for the tip!